src/app/components/elements/HelpTip/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+export default class HelpTip extends React.Component {
+    static propTypes = {
+        children: PropTypes.any.isRequired,
+        content: PropTypes.oneOfType([PropTypes.string, PropTypes.array])
+            .isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            visible: false,
+        };
+    }
+
+    show = () => this.setVisibility(true);
+    hide = () => this.setVisibility(false);
+
+    setVisibility = visible => {
+        this.setState(
+            Object.assign({}, this.state, {
```

Comment:
Is this needed? I thought react's setState would do something similar behind the scenes

Reply:
You're right, it's not needed.